Workaround autoreconf removing the setting of $(GETTEXT_PACKAGE) in
libvisual-0.4.0/po/Makefile.in.in

--- libvisual-0.4.0/po/Makevars	2016-10-07 10:52:34.532032488 -0700
+++ libvisual-0.4.0/po/Makevars	2016-10-07 10:52:48.854073483 -0700
@@ -1,7 +1,7 @@
 # Makefile variables for PO directory in any package using GNU gettext.
 
 # Usually the message domain is the same as the package name.
-DOMAIN = $(GETTEXT_PACKAGE)
+DOMAIN = libvisual-0.4
 
 # These two variables depend on the location of this directory.
 subdir = po
